src/cli.js

```diff
@@ -13,6 +13,10 @@ function getArgs() {
     )
     .describe('className', 'Create a React component (wraps JSX in React.createClass call)')
     .alias('className', 'c')
+    .boolean('e')
+    .describe('e', 'Decide whether to wrap in module.exports')
```

Comment:
This option seems useful, but it's not really relevant to this pull request. Could you please split it out into a separate one? I think because you created this pull request off your master branch, Github is including all the changes on your master branch. You should make a separate branch with just the changes you want to submit :)